src/main/java/edu/ucsb/cs56/ucsb_courses_search/service/UserMembershipService.java

```diff
@@ -20,30 +21,76 @@
 
     private Logger logger = LoggerFactory.getLogger(UserMembershipService.class);
 
+    @Value("${app.admin.emails:}")
+    final private List<String> adminEmails = new ArrayList<String>();
+
+    @Value("${app.member.hosted-domain}")
+    final private String memberHostedDomain = "ucsb.edu";
```

Comment:
Currently, this value is hard coded to ucsb.edu, and the @value() is not inputting the value from application.properties. This should be logged as a bug and fixed later